### PR TITLE
chore(FX-4197, FX-4195): activity panel improvements

### PR DESCRIPTION
**Pull request by dimatretyak**
The type of this PR is: **Chore**

This PR solves [FX-4197], [FX-4195]

Review app: https://activity-panel-improvements.artsy.net/

### Changes
* Added hover states
* Made tabs sticky to the top

#### Dropdown mode (desktop)
https://user-images.githubusercontent.com/3513494/188157628-960242ef-2ed4-4225-b695-550d43573944.mp4

#### Page mode (mWeb)
https://user-images.githubusercontent.com/3513494/188157720-91737225-35bd-410d-b4c8-e371ae1ba351.mp4

[FX-4197]: https://artsyproduct.atlassian.net/browse/FX-4197?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[FX-4195]: https://artsyproduct.atlassian.net/browse/FX-4195?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by gkartalis — APPROVED** [review]
💯 great work! 🔥 Left one comment but up to you

File: src/Components/Notifications/NotificationsTabs.tsx [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+            flex={1}
+          >
+            <NotificationBaseTab active={i === activeTabIndex} variant="sm">
+              <span>{tab.child.props.name}</span>
+            </NotificationBaseTab>
+          </Clickable>
+        )
+      })}
+    </BaseTabs>
+  )
```
Comment (gkartalis) [inline src/Components/Notifications/NotificationsTabs.tsx:39]: **suggestion(non-blocking):** How do you feel about moving this to its own component? 🤔

Comment (dimatretyak) [reply src/Components/Notifications/NotificationsTabs.tsx:39]: I've been thinking about that too. But I didn't find any advantages that these changes would give us (the component is used in one place, it is necessary to duplicate the TS types for props)

Comment (gkartalis) [reply src/Components/Notifications/NotificationsTabs.tsx:39]: consider this as resolved then

**Review by dblandin — APPROVED** [review]
LGTM!

**Pull request merged**